### PR TITLE
Feat: Added Alert component

### PR DESCRIPTION
**Pull request by sushil-mod**

### CODE REVIEW

File: Components/alert.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+.alert{
+    width: 30rem;
+    border:4px solid var(--alert-color) ;                              
+    background-color: var( --alert-bg-color);
+} 
+.alert-icon{
+    height: 50px;
+    width: 50px;
```
Comment (ozairkhan2810) [inline Components/alert.css]: use rem

**Review by ozairkhan2810 — APPROVED** [review]
gave one suggestion then can merge

Also add description while raising PR so that it will be easier for reviewer

**Review by shaksham08 — APPROVED** [review]
Looks good just follow the comments

File: Components/alert.css (hunk 1 above)
Comment (sushil-mod) [reply Components/alert.css]: done, changed to 3rem

**Pull request merged**